.githooks/pre-commit

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+lerna run lint
```

Comment:
Does this require to have lerna globally installed? Does it work with local lerna as well?